railties/test/application/rake_test.rb

```diff
@@ -129,6 +129,8 @@ def test_rake_routes_calls_the_route_inspector
         end
       RUBY
 
+      add_to_config "config.action_cable.routes_enabled = false"
```

Comment:
It doesn't do anything special, it only prepend the route so if you add a new route with a different controller it will be override